electron/src/util/productInfoLoader.ts

```diff
@@ -0,0 +1,16 @@
+"use strict";
```

Comment:
As name suggests. A loader for the product.json file of the app, which contains target-specific info (just like pxtarget.json), such as the name of the app, the app GUID, etc